network.go

```diff
@@ -23,6 +23,26 @@ const (
 	NetworkSimnet
 )
 
+// NetworkFromStr parses a network from string.
+func NetworkFromStr(text string) (Network, error) {
+	switch text {
+	case "mainnet":
+		return NetworkMainnet, nil
+
+	case "testnet":
```

Comment:
Add `testnet3` as well to be compliant with some notations?

Reply:
It doesn't match the network returned by `lnd.GetInfo`. Don't really want to add anything we don't need now.